api/src/main/resources/workbench-api.yaml

```diff
@@ -5855,6 +5863,23 @@ definitions:
     properties:
       runtime:
         "$ref": "#/definitions/Runtime"
+  PersistentDiskRequest:
+    description: configuration to create a disk
+    type: object
+    required:
+      - name
+    properties:
+      name:
```

Comment:
Actually, for this creating runtime with pd scenario, we would replace this given param "name" with a unique variable  "all-of-us-pd-" + getUserId(). Do you mean by that?

Reply:
Understood. So my PD name might be `all-of-us-pd-22`. Each workspace is its own project, and has its own runtimes, so I'm assuming there will not be a conflict if I create `all-of-us-pd-22` in workspace1, and then I again create `all-of-us-pd-2` in workspace2, i.e. they are not _globally_ unique. It's pretty safe to assume this will work, but something that's worth checking once we're in a position to test this out.

Reply:
I think most compute resources are not globally unique. 
https://cloud.google.com/compute/docs/naming-resources#name_uniqueness
+1 safe for now, and we can easily test it in UI by just creating 2 PDs in different workspace. 

One more thing to consider is: if I delete a PD then recreate one within the same workspace, will that cause naming conflict?  Not sure the deletion life cycle of a PD, and how long it take for GCP to actually remove it. Something else worth to test.

Reply:
I found the CRUD APIs for PD in Leo all require both googleProject(equivalent to workspace for AoU) and PDName(equivalent to userID for AoU) params. It's a reasonable assumption that Leo will create unique globally unique PD in namespace <googleProject, PDName>. That meets our need of globally unique PD in namespace <workspace, userID>. But that's a guess anyway. I'll verify it.